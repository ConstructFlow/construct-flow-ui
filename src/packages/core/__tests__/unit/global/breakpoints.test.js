const sass = require('sass');

describe('Colors variables', () => {
  const result = sass
    .compile('./src/packages/core/__tests__/fixtures/breakpoints.fixtures.scss')
    .css.toString();

  it('Should has correct query for breakpoint', async () => {
    expect(result).toMatchSnapshot();
    expect(result).toContain('@media (min-width: 0)');
    expect(result).toContain('@media (min-width: 480px)');
    expect(result).toContain('@media (min-width: 720px)');
    expect(result).toContain('@media (min-width: 720px)');
    expect(result).toContain('@media (min-width: 960px)');
    expect(result).toContain('@media (min-width: 1200px)');
    expect(result).toContain('@media (min-width: 1400px)');
  });
});
