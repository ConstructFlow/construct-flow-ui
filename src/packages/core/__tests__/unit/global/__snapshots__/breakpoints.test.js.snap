// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Colors variables Should has correct query for breakpoint 1`] = `
"@media (min-width: 0) {
  .testBreakpoint__xs {
    color: red;
  }
}

@media (min-width: 480px) {
  .testBreakpoint__sm {
    color: blue;
  }
}

@media (min-width: 720px) {
  .testBreakpoint__md {
    color: green;
  }
}

@media (min-width: 960px) {
  .testBreakpoint__lg {
    color: yellow;
  }
}

@media (min-width: 1200px) {
  .testBreakpoint__xl {
    color: aqua;
  }
}

@media (min-width: 1400px) {
  .testBreakpoint__specific-size {
    color: black;
  }
}"
`;
